filebeat/docs/modules/okta.asciidoc

```diff
@@ -0,0 +1,27 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-okta]]
+[role="xpack"]
+
+:modulename: okta
+:has-dashboards: false
+
+== OKTA module
```

Comment:
I think it should be "Okta" everywhere the product name is needed. I don't see "OKTA" in use anywhere on their website.